web/templates/topic_list_item.hbs

```diff
@@ -1,12 +1,15 @@
-<li class='bottom_left_row {{#if is_active_topic}}active-sub-filter{{/if}} {{#if is_zero}}zero-topic-unreads{{/if}} {{#if is_muted}}muted_topic{{/if}} {{#if is_unmuted}}unmuted_topic{{/if}} topic-list-item' data-topic-name='{{topic_name}}'>
+<li class='bottom_left_row {{#if is_active_topic}}active-sub-filter{{/if}} {{#if is_zero}}zero-topic-unreads{{/if}} {{#if is_muted}}muted_topic{{/if}} {{#if is_unmuted_or_followed}}unmuted_or_followed_topic{{/if}} topic-list-item' data-topic-name='{{topic_name}}'>
     <span class='topic-box'>
         <span class="sidebar-topic-check">
             {{topic_resolved_prefix}}
         </span>
         <a href='{{url}}' class="topic-name" title="{{topic_name}}">
             {{topic_display_name}}
         </a>
-        {{#if contains_unread_mention}}
+        {{#if is_followed}}
+            <i class="zulip-icon zulip-icon-follow" aria-hidden="true"> </i>
+            &nbsp;
+        {{else if contains_unread_mention}}
```

Comment:
I think the discussion was that we wanted to have `contains_unread_mention` take precedence, right?

Reply:
Swapped this as preparation for merging.